app/ante/eth.go

```diff
@@ -39,7 +39,10 @@ func NewEthSigVerificationDecorator(ek EVMKeeper) EthSigVerificationDecorator {
 // Failure in RecheckTx will prevent tx to be included into block, especially when CheckTx succeed, in which case user
 // won't see the error message.
 func (esvd EthSigVerificationDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate bool, next sdk.AnteHandler) (newCtx sdk.Context, err error) {
-	chainID := esvd.evmKeeper.ChainID()
+	chainID, err := ethermint.ParseChainID(ctx.ChainID())
+	if err != nil {
+		return ctx, sdkerrors.Wrapf(err, "invalid chain-id %s", ctx.ChainID())
```

Comment:
The method `ethermint.ParseChainID` being used multiple times in project, almost will panic when error, and one will returns `sdkerrors.Wrapf(err, "failed to parse chainID: %s", signerData.ChainID)` so I think we should use the same message here and there to make err msg content more consistency

Reply:
err msg changed.